setup.sh

```diff
@@ -5,7 +5,7 @@ set -o nounset
 set -o pipefail
 
 HELM=${HELM:-helm}
-VVP_CHART=${VVP_CHART:-ververica/ververica-platform}
+VVP_CHART=${VVP_CHART:-ververica-platform}
```

Comment:
This var may not be useful for testing a local chart anymore, since `--repo` might require that the chart is pulled from a remote source. (i.e. it will look for a chart named `my-chart.tgz` in the provided repo)

I'd suggest either removing this var, or defaulting it to nothing (`VVP_CHART=${VVP_CHART:-}`) and then in `helm_install_vvp()`, have different `helm_install` invocation for `[ -n "$VVP_CHART" ]`.

Reply:
Good point. I went with the second proposal.